### PR TITLE
removing <p> and </p> tags from json scraper

### LINKED ISSUE #16: Fix html tags on individual listing show page

### PR DESCRIPTION
**Pull request by georgeteo**
fixes #16 

I think paragraph tags are the only things that are showing up in the description so that is all I removed.

### CODE REVIEW

**Comment by fatlotus** [comment]
I think there are also <a href> tags. Also, does this properly take care of newlines?

**Comment by georgeteo** [comment]
I left the new line characters inside. I think they'll be rendered correctly? 

Also what other tags are you referring to?

**Comment by fatlotus** [comment]
Ah, dang, I had a `<a href>` but Markdown eated it.

**Comment by fatlotus** [comment]
Other than that looks good to me.

**Comment by georgeteo** [comment]
Ok. Have fixed it,

File: live_migration.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def pull_from_listing(permalink):
         key_name=json_data["permalink"],
         seller=json_data["seller"]["email"],
         posting_time=posting_time,
-        description=json_data["description"],
+        description=re.sub(r'(<a.*>\n*)', '', json_data["description"].replace("<p>", "").replace("</p>", ""))
+        .replace,
```
Comment (fatlotus) [inline live_migration.py]: Derp... I did a bad job reviewing this. The .replace is a syntax error.

**Pull request merged**